src/main/java/codeu/model/store/basic/DefaultDataStore.java

```diff
@@ -98,7 +98,7 @@ private void addRandomUsers() {
     Collections.shuffle(randomUsernames);
 
     for (int i = 0; i < DEFAULT_USER_COUNT; i++) {
-      User user = new User(UUID.randomUUID(), randomUsernames.get(i), Instant.now());
+      User user = new User(UUID.randomUUID(), randomUsernames.get(i), "password", Instant.now());
```

Comment:
Can you put "password" into a constant, and make it public so that the tests can use it as well?

Reply:
Yes I can! I will commit the and push that change soon.

Reply:
Looking back at the instructions, it says to "supply a hard-coded password" for that part. Should I still make it into a constant?